src/app/zap-templates/templates/app/helper.js

```diff
@@ -448,10 +448,22 @@ async function zapTypeToClusterObjectType(type, isDecodable, options)
     }
 
     if (types.isEnum) {
+      // Catching baseline enums and converting them into 'uint[size]_t'
+      let s = type.toLowerCase().match(/^enum\d+$/g) ? type.match(/\d+$/g).join('') : null
+      if (s)
+      {
+        return 'uint' + s + '_t'
```

Comment:
```suggestion
      let s = type.toLowerCase().match(/^enum(\d+)$/g);
      if (s)
      {
        return 'uint' + s[1] + '_t'
```

Reply:
> Suggested change 
>       let s = type.toLowerCase().match(/^enum\d+$/g) ? type.match(/\d+$/g).join('') : null
>       if (s)
>       {
>         return 'uint' + s + '_t'
>       let s = type.toLowerCase().match(/^enum(\d+)$/g);
>       if (s)
>       {
>         return 'uint' + s[1] + '_t'

So the matches are different `match(/^enum\d+$/g)` and `type.match(/\d+$/g)`

Reply:
Yes... but so what?  My proposed code above extracts the trailing `\d+` as the first parenthesized match, which is why it ends up in `s[1]`.

Reply:
Ah, you want to drop the "g" flag is what you want.